lib/money.ex

```diff
@@ -2495,6 +2497,17 @@ defmodule Money do
     nil
   end
 
+  defp symbol_script(symbols, locale) do
+    script =
+      locale
+      |> Cldr.Locale.script_from_locale()
```

Comment:
That you have to go through this atom/string munging is horrible. I need to update `ex_cldr_numbers` to use a canonical form, which will be `:Latn`. I will fix that in time for the next `ex_cldr` release cycle which will be April 17th alongside CLDR 45's release.

Reply:
Perfect

Reply:
Actually I created some confusion between a script and a number system. `:Latn` is the script. `:latn` is a number system. So here the call should be `Cldr.Number.System.number_system_from_locale/1` which will return `:arab` for the locale `ar-EG` as expected.  I'll do that on my end, no action required on your end. And this will get rid of the horrible atom/string/atom munging.

Reply:
@kipcole9 sorry I must have confused both .. perfect .. thank you for the solution